app/models/response.rb

```diff
@@ -23,7 +23,7 @@ def team_has_user?(user)
     reviewer.team.has_user user
   end
 
-  def display_as_html(prefix = nil, count = nil, _file_url = nil)
+  def display_as_html(prefix = nil, count = nil, _file_url = nil, feedback_html = nil)
```

Comment:
Why did you make `feedback_html` as a parameter? It is weird for me to concatenate html in the view.